api/kessel/inventory/v1beta1/k8s_clusters_service.proto

```diff
@@ -15,10 +15,7 @@ message CreateK8sClusterRequest {
   K8sCluster k8s_cluster = 1;
 }
 
-message CreateK8sClusterResponse {
-  // The K8s cluster created in Kessel Asset Inventory
-  K8sCluster k8s_cluster = 1;
-}
+message CreateK8sClusterResponse {}
```

Comment:
Asked in the previous PR:

About the requirement of not returning anything in the create requests.
Wouldn't this be an issue for the caller if they want to do any subsequent call with the newly created resource?
e.g. getting the id of the newly created resource for any following up process to initialize or prepare anything else?

Reply:
If you look at the update/delete interfaces, they leverage the reporter information, not the "internal" ID.  Thus, they do not need to get the ID returned.